cupy/fallback_mode/__init__.py

```diff
@@ -0,0 +1,12 @@
+# Attributes and Methods for fallback_mode
+# Auto-execute numpy method when corresponding cupy method is not found
+
+# "NOQA" to suppress flake8 warning
+from cupy.fallback_mode import fallback # NOQA
```

Comment:
Minor nit: I don't think this import is used (see below) :-)

Reply:
Thanks! It's resolved now.